models/issue.rb

```diff
@@ -110,6 +231,16 @@ def get_all_comments
       all
    end
 
+   def due_on
+      time = @issue[:milestone][:due_on]
```

Comment:
Is there a reasonable guarantee that `@issue[:milestone]` will not be `nil` (or another non-`Array` value)